config/example_config.yaml

```diff
@@ -102,7 +102,7 @@ blocktx:
       maxOpenConns: 80 # maximum open connections
       sslMode: disable
   recordRetentionDays: 28 # number of days for which data integrity is ensured
-  registerTxsInterval: 10s # time interval to read from the channel registered transactions
+  requestTxsInterval: 10s # time interval to register and request mined transactions
```

Comment:
I think we should keep here the name `registerTxsInterval` and also topic `RegisterTxTopic = "register-tx"` and drop `requestTxsInterval` instead because a new tx is always registered. So that is kind a core functionality.

The functionality for requesting a tx because it wasn't mined for too long is rather an exception in case that e.g. `blocktx` was failing for some reason. So that is like an auxiliary functionality.